bin/perlmutter-install.sh

```diff
@@ -39,12 +21,12 @@ rm -rf ${FIRECROWN_DIR}
 
 # Add some things that are set up automatically
 cat >  ./conda/etc/conda/activate.d/activate-txpipe.sh <<EOF
-module load cray-mpich-abi
+module load mpich/4.2.2
```

Comment:
Is this getting loaded twice? line 24 and line 8?

Reply:
The one on line 24 is not being executed here, it's being written to `./conda/etc/conda/activate.d/activate-txpipe.sh`.  I've added a comment to explain, and made it clearer with indentation.